### ISSUE TITLE
 android:exported needs to be explicitly specified forPaymentDetailsActivity

### ISSUE THREAD

**Issue by DeepakHeadspire** [Instamojo/android-sdk#111]
version : 3.6.0
have issue related to manifest  for android 12 and later version 

Caused by: java.lang.RuntimeException: Manifest merger failed : android:exported needs to be explicitly specified for element <activity#com.instamojo.android.activities.PaymentDetailsActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.

and  the latest version 3.7.1 having gradle build issue with error
Internal error: Artifact type null not expected, only jar or aar are handled.

**Comment by Prafulapuri** [comment]
Please Do something it's not working

**Comment by SriramRajendranR** [comment]
yes, build issue is fixed by adding this -  implementation ('com.instamojo:android-sdk:3.7.1@aar') { transitive = true }
